src/index.js

```diff
@@ -1,4 +1,4 @@
-export { default as Controls, Compass, ZoomControls } from './Controls'
-export { default as Map, connectToMap, updateMapFromUrl, updateUrlFromMap } from './Map'
-export { Popup, PopupBase, PopupDefaultInsert } from './Popup'
+export * from './Controls'
+export * from './Map'
+export * from './Popup'
```

Comment:
here's a key difference- we will no longer need to list utils individually, they will all get caught by `*`